src/main/java/eu/dissco/backend/service/SpecimenService.java

```diff
@@ -43,39 +45,62 @@ public class SpecimenService {
       "ods", "http://github.com/DiSSCo/openDS/ods-ontology/terms/",
       "hdl", "https://hdl.handle.net/",
       "dcterms", "http://purl.org/dc/terms/");
+  private final Map<String, String> paramMapping = createParamMapping();
   private final ObjectMapper mapper;
   private final SpecimenRepository repository;
   private final ElasticSearchRepository elasticRepository;
   private final DigitalMediaObjectService digitalMediaObjectService;
   private final AnnotationService annotationService;
   private final MongoRepository mongoRepository;
 
+  private static Map<String, String> createParamMapping() {
+    var map = new HashMap<String, String>();
+    map.put("country", "digitalSpecimen.ods:attributes.dwc:country.keyword");
+    map.put("countryCode", "digitalSpecimen.ods:attributes.dwc:countryCode.keyword");
+    map.put("physicalSpecimenId", "digitalSpecimen.ods:physicalSpecimenId.keyword");
+    map.put(MIDS_LEVEL, MIDS_LEVEL);
+    map.put("typeStatus", "digitalSpecimen.ods:attributes.dwc:typeStatus.keyword");
+    map.put("license", "digitalSpecimen.ods:attributes.dcterms:license.keyword");
+    map.put("hasMedia", "digitalSpecimen.ods:attributes.ods:hasMedia.keyword");
+    map.put("organisationId", "digitalSpecimen.ods:attributes.ods:organisationId.keyword");
+    map.put("organisationName", "digitalSpecimen.ods:attributes.ods:organisationName.keyword");
+    map.put("sourceSystemId", "digitalSpecimen.ods:attributes.ods:sourceSystemId.keyword");
+    map.put("type", "digitalSpecimen.ods:type.keyword");
+    map.put("specimenName", "digitalSpecimen.ods:attributes.ods:specimenName");
+    map.put("q", "q");
+    return map;
+  }
```

Comment:
The mapping is some extra work, but I think this will make it much cleaner to work with the API